sys/linux/socket_netlink_route.txt

```diff
@@ -237,6 +245,49 @@ rtm_mpls_policy [
 	RTA_TTL_PROPAGATE	nlattr[RTA_TTL_PROPAGATE, int8]
 ] [varlen]
 
+fib_rule_uid_range {
+	start	int32
+	end	int32
+}
+
+fib_rule_port_range {
+	start	int16
+	end	int16
+}
+
+fra_generic_policy [
+	FRA_IIFNAME		nlattr[FRA_IIFNAME, devname]
+	FRA_OIFNAME		nlattr[FRA_OIFNAME, devname]
+	FRA_PRIORITY		nlattr[FRA_PRIORITY, int32]
+	FRA_FWMARK		nlattr[FRA_FWMARK, int32]
+	FRA_FWMASK		nlattr[FRA_FWMASK, int32]
+	FRA_TABLE		nlattr[FRA_TABLE, int32]
+	FRA_SUPPRESS_PREFIXLEN	nlattr[FRA_SUPPRESS_PREFIXLEN, int32]
+	FRA_SUPPRESS_IFGROUP	nlattr[FRA_SUPPRESS_IFGROUP, int32]
+	FRA_GOTO		nlattr[FRA_GOTO, int32]
+	FRA_L3MDEV		nlattr[FRA_L3MDEV, int8]
+	FRA_UID_RANGE		nlattr[FRA_UID_RANGE, fib_rule_uid_range]
+	FRA_PROTOCOL		nlattr[FRA_PROTOCOL, int8]
```

Comment:
Seems this this needs to be flags with RTPROT_* constants from uapi/linux/rtnetlink.h